conf/messages.en

```diff
@@ -115,18 +115,18 @@ empref.change.hidden = the Employer PAYE Reference (EMPREF) for {0}
 hasSecondaryContact.title = Is there someone else we can contact if the first person or team is not available?
 hasSecondaryContact.heading = Is there someone else we can contact if {0} is not available?
 hasSecondaryContact.checkYourAnswersLabel = Is there someone else we can contact if {0} is not available?
-hasSecondaryContact.error.required = Select yes if there is someone else we can contact if {0} is not available
+hasSecondaryContact.error.required = Select yes if there is someone else we can contact
```

Comment:
since you have removed the param passed into this messge key pair, then will also need to update:
https://github.com/hmrc/digital-platform-reporting-operator-frontend/blob/main/app/forms/HasSecondaryContactFormProvider.scala